cypress/support/index.js

```diff
@@ -0,0 +1,62 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
+
+beforeEach(() => {
+    cy.visit('http://localhost:8000')
+
+    cy.url().then(url => {
+        if (url.includes('setup_admin')) {
+            cy.get('#inputCompany')
+                .type('company')
+                .should('have.value', 'company')
+
+            cy.get('#inputName')
+                .type('name')
+                .should('have.value', 'name')
+
+            cy.get('#inputEmail')
+                .type('fake@email.com')
+                .should('have.value', 'fake@email.com')
```

Comment:
If we change this to @posthog.com it'll automatically be ignored in our real_users cohort